android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCApiClient.kt

```diff
@@ -1,20 +1,19 @@
 package com.devcycle.sdk.android.api
 
-import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.google.gson.Gson
 import okhttp3.OkHttpClient
 import retrofit2.Retrofit
-import retrofit2.converter.jackson.JacksonConverterFactory
+import retrofit2.converter.gson.GsonConverterFactory
 
 internal class DVCApiClient {
     private val okBuilder: OkHttpClient.Builder = OkHttpClient.Builder()
-    private val objectMapper = jacksonObjectMapper()
     private val adapterBuilder: Retrofit.Builder = Retrofit.Builder()
-        .addConverterFactory(JacksonConverterFactory.create(objectMapper))
 
-    fun initialize(baseUrl: String): DVCApi {
+    fun initialize(baseUrl: String, gsonMapper: Gson): DVCApi {
```

Comment:
is this a required field? why is this needed on the `initialize` function?

Also, would this require this change to be a major version?